Cargo.toml

```diff
@@ -15,11 +15,7 @@ default-members = [
     "pricegraph",
 ]
 
-[profile.release]
-opt-level = "s"
-
 [patch.crates-io]
 # Needed to make coverage work on MacOS (https://github.com/servo/core-foundation-rs/pull/357)
 core-foundation = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
 core-foundation-sys = { git = "https://github.com/servo/core-foundation-rs", tag="v0.20.1"}
-
```

Comment:
There is still an empty line here btw (there were two).

Reply:
Hahaha, don't worry, there is a prominent :no_entry_sign: icon when there is a missing newline at the end of file. There is none here :smile: